doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -60,6 +60,12 @@ Options:
 
  Force create sandbox and overwrite existing directory 
 
+ 4. relocate flag for creating: ::
```

Comment:
Please make this a full sentence and explain what it means to relocate something in the context.